package.json

```diff
@@ -34,6 +34,7 @@
     "chromecast-player": "^0.2.3",
     "debounced-seeker": "^1.0.0",
     "debug": "^2.1.0",
+    "diveSync": "^0.3.0",
```

Comment:
It is in order to use [this](https://github.com/xat/castnow/pull/186/files/b59a941aed77e2b4edc76327421dcf7b43bd821c#diff-8a1d65749fc58007cb0a97f5050c5f24R31), which is a straightforward way for listing files recursively or non-recursively just by setting a flag.

Doing the same just with `fs-extended` would lead to more complex code. Moreover, `fs-extended` is deprecated, and its author points to `fs-extra`, which does not support filtering. So I think that it would be desirable not only to add `diveSync` as a requirement, but also to rewrite `isDir()` to completely remove `fs-extended`.